monix-reactive/jvm/src/test/scala/monix/reactive/compress/InflateTest.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2014-2020 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.reactive.compress
+
+import minitest.api.AssertionException
+import monix.reactive.Observable
+
+object InflateTest extends BaseTestSuite with DeflateTestUtils {
```

Comment:
I feel like we could reuse a lot of test cases between `InflateTest` and `GunzipTest` - do you think it's worth it?

Reply:
Good point, done.